components/x-gift-article/src/SharedLinkTypeSelector.jsx

```diff
@@ -70,14 +75,29 @@ export const SharedLinkTypeSelector = (props) => {
 						href={`${enterpriseEnabled ? 'mailto:customer.success@ft.com' : 'mailto:help@ft.com'}`}
 						rel="noreferrer"
 						target="_blank"
-						data-trackable="enterprise-out-of-credits"
-					>
+						data-trackable="enterprise-out-of-credits">
 						contact support
 					</a>
 					.
 				</NoCreditAlert>
 			)}
 			{showAdvancedSharingOptions && <AdvancedSharingOptions {...props} />}
+			{hasHighlights && advancedSharingEnabled && (
```

Comment:
I believe this should actually be `enterpriseEnabled` and not `advancedSharingEnabled` as we want _all_ b2b users to share highlights, not just advanced sharing users

Reply:
so the `advancedSharingEnabled` checks if `enterpriseEnabled` & `enterpriseRequestAccess` which i don't exactly know what does it mean , i will check  if it is only related to advance sharing , I will update the code to only check for `enterpriseEnabled`